FirebaseSegmentation/Sources/Public/FIRSegmentation.h

```diff
@@ -25,7 +25,6 @@ NS_ASSUME_NONNULL_BEGIN
  * current app installation for segmentation purposes. If the custom installation identifier is
  * explicitely set to nil, any existing custom installation identifier data will be removed.
  */
-NS_SWIFT_NAME(Segmentation)
 @interface FIRSegmentation : NSObject
```

Comment:
This `NS_SWIFT_NAME` and the one below for `SegmentationErrorCode` are still required.